jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartTest.java

```diff
@@ -659,6 +619,22 @@ public void testBadHeaderNames(String badHeader)
         assertThat(listener.failure.getMessage(), containsStringIgnoringCase("invalid header name"));
     }
 
+    @Test
+    public void testEncodeContentDispositionFilename()
+    {
+        assertThat(MultiPart.encodeContentDispositionFileName("test.xml"), is("filename=\"test.xml\""));
+        assertThat(MultiPart.encodeContentDispositionFileName("test ✓.xml"), is("filename=\"test _.xml\"; filename*=UTF-8''test%20%E2%9C%93.xml"));
+        assertThat(MultiPart.encodeContentDispositionFileName("test ✓.xml", StandardCharsets.UTF_16), is("filename=\"test _.xml\"; filename*=UTF-16''╆䔥䙆┰ぴ┰づ┰び┰ぴ┰〫┲㜥ㄳ┰〮┰へ┰ね┰ぬ"));
```

Comment:
Looking at UTF-16 in the raw like this is so weird.

Reply:
yeah,  I think it is OK to have like that.... but it is strange.